guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -279,7 +279,7 @@ public Response createProject(
             throw new WebApplicationException( Response.status( Response.Status.NOT_FOUND ).entity( repositoryName ).build() );
         }
 
-        Set<Project> projects = projectService.getProjects(repository, repository.getCurrentBranch());
+        Set<Project> projects = projectService.getProjects(repository, "master");
```

Comment:
Yes. Based on my investigations it only uses master. I'm not sure this should use the repository.getDefaultBranch(). The problem with that is, if there is no master it just returns the first branch there is and that is probably not what is wanted here.